.github/workflows/ci-containers.yaml

```diff
@@ -60,11 +60,23 @@ jobs:
             type=ref,event=branch
             type=ref,event=pr
 
-      - name: Build the ${{ matrix.container }} image
+      - name: Build the service image
         id: docker_build
         uses: docker/build-push-action@v4
         with:
-          context: ${{ matrix.container }}
+          context: service
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
Yeah so the image name used is built from the matrix.container value so will match that.
You will need to update either the deployment to match (which is probably easiest) or provide a tag at line 52 using an appropriate value: https://github.com/calyptia/vivo/blob/1f6c929b93fcc69a6d05a0b4ca38ef7db6480acf/.github/workflows/ci-containers.yaml#L52

Why not move the service image as well under packages so you do not need that prefix? 
You could also do some processing of the matrix value but it all seems a bit brittle to me.
Otherwise split it into two jobs, one building the service image and one build the frontend one but it means duplicating steps which is why I added a matrix job.